### PR TITLE
[ADD] Added website_seo_public_category

### PR DESCRIPTION
**Pull request by wpichler**
This module extends the functionality of the module website_seo_redirection better support product.public.category.
- Does support rename of public category
- Does rewrite the website_url to deliver the seo url

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hello @wpichler ! Thanks for this contribution.

I don't know how to test it. Please could you add to the usage section how can I test this module in runbot please? https://github.com/OCA/website/pull/522/files#diff-4e091c6cc99b8e68eb94d01ab846edc3R3

Also you should review runbot :red_circle: and travis error please :red_circle: 

Ping me when done please. Thanks! :smile:

**Comment by rafaelbn** [comment]
Ping @cristinamartinrod @Yajo @chienandalu

File: website_seo_public_category/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+* Got to Website Admin -> Configuration -> eCommerce -> Website PRoduct Categories
+* Create a Product Categorie
```
Comment (chienandalu) [inline website_seo_public_category/readme/USAGE.rst:4]: s/categorie/category

File: website_seo_public_category/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+* Product Categorie will be available in shop under the path /shop/categorie/name-ID
```
Comment (chienandalu) [inline website_seo_public_category/readme/USAGE.rst:5]: s/categorie/category

**Review by chienandalu — CHANGES_REQUESTED** [review]
Couldn't make it work in runbot:

#. Created a redirection
#. Changed the category name
#. The change wasn't propagated to the redirection...

Review minor LINT errors as well

**Pull request closed without merging**